src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.rootLogger=DEBUG, STDOUT, file
```

Comment:
De debug statements lijkt communicatie te loggen tussen het project en de chromedriver. In dit geval zou ik dergelijke informatie niet loggen, want dat biedt geen toegevoegde waarde voor ons. We kunnen er in 99,9% van de gevallen vanuit gaan dat wanneer er problemen ontstaan we deze niet vanuit deze communicatie op gaan lossen, maar eerder vanuit degelijke error logging die de webdriver terug geeft. Hier zie ik niet echt de toegevoegde waarde van.